qa/rpc-tests/remove_sprout_shielding.py

```diff
@@ -129,7 +153,7 @@ def run_test (self):
         self.sync_all()
 
         # Create z_mergetoaddress Sprout -> Sprout transaction on node 1. Should pass
-        merge_tx_2 = self.nodes[1].z_mergetoaddress(["ANY_SPROUT"], sprout_addr_node2)
+        merge_tx_2 = self.nodes[1].z_mergetoaddress(["ANY_SPROUT"], n2_sprout_addr)
```

Comment:
Nonblocking: this change and some of those above are unnecessary here, it probably leaked in from my code, but it's fine.

Reply:
I’m inclined to leave them in … they improve the clarity & consistency, and some of them are entwined with actual changes.